Source/Outposts/Outpost/Outpost.cs

```diff
@@ -148,7 +164,11 @@ public virtual void Produce()
 
         public override void SpawnSetup()
         {
-            base.SpawnSetup();
+            base.SpawnSetup();            
+            if (deliveryMap == null)
+            {
+                deliveryMap = Find.Maps.Where(m => m.IsPlayerHome).OrderByDescending(m => Find.WorldGrid.ApproxDistanceInTiles(m.Parent.Tile, Tile)).FirstOrDefault();
```

Comment:
```suggestion
                deliveryMap = Find.Maps.Where(m => m.IsPlayerHome).OrderBy(m => Find.WorldGrid.ApproxDistanceInTiles(m.Parent.Tile, Tile)).FirstOrDefault();
```
Since we're getting the first and we want the smallest, this will put the smallest first.